docs/solutions/reference-designs/eval-cn0584-ebz/HDL-project/index.rst

```diff
@@ -0,0 +1,100 @@
+the ADC path. The clock signals are generated by the AXI_PWM_GEN and AXI_CLKGEN
+IP cores.
+
+DAC Interface
+-------------
+
+The DAC data may be sourced from an internal data generator (DDS or pattern),
+from the external DDR via DMA or from the ADC. The source control selection can
+be made by updating the :dokuwiki:`REG_CHAN_CNTRL_7 (DAC Channel) </resources/fpga/docs/hdl/regmap#:~:text=DAC%20Channel%20(axi_ad*)>`
+register.
+
+ADC Interface
+-------------
+
+The ADC data is sent to DDR via DMA. The ADC PACK IP (util_cpack) allows the use
+of a single DMA IP core for all four analog-to-digital converters.
+
+Control and SPI
+---------------
+
+The SPI signals of the ADC are controlled by the SPI port of the ARM processor.
+
+The DAC’s control is performed by the :git-repo:`axi_ad35552r </analogdevicesinc/hdl/tree/master/library/axi_ad3552r>`
+IP core.
+
+Download
+--------
+
+.. admonition:: Download
+
+    The HDL repository, list of supported carriers, and the list of required IP
+    cores can be found here:
+
+    - Hardware: :adi:`EVAL-CN0585-FMCZ <cn0585>`
+    - HDL Project: `hdl_cn0585 <https://github.com/analogdevicesinc/hdl/tree/cn0585_v1/projects/cn0585_fmcz>`_
+    - Carriers: `zed <https://github.com/analogdevicesinc/hdl/tree/cn0585_v1/projects/cn0585_fmcz/zed>`_
+    - Library Cores:
+
+            - `axi_clkgen <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_clkgen>`_
+            - `axi_dmac <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_dmac>`_
+            - `axi_hdmi_tx <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_hdmi_tx>`_
+            - `axi_i2s_adi <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_i2s_adi>`_
+            - `axi_ltc2387 <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_ltc2387>`_
+            - `axi_ad3552r <http://github.com/analogdevicesinc/hdl/tree/cn0585_v1/library/axi_ad3552r>`_
+            - `axi_pwm_gen <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_pwm_gen>`_
+            - `axi_spdif_tx <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_spdif_tx>`_
+            - `axi_sysid <http://github.com/analogdevicesinc/hdl/tree/master/library/axi_sysid>`_
+            - `sysid_rom <http://github.com/analogdevicesinc/hdl/tree/master/library/sysid_rom>`_
+
+Reference Demos & Software
+--------------------------
+
+- :adi:`CN0584 Precision Low Latency Development Kit <cn0584>`
+- :adi:`CN0585 Quad Channel, Low Latency, Data Acquisition and Signal Generation Module <cn0585>`
+- :dokuwiki:`AXI_AD3552R IP core </resources/fpga/docs/axi_ad3552r>`
+- :dokuwiki:`AXI_LTC2387 IP core </resources/fpga/docs/axi_ltc2387>`
+- :dokuwiki:`AD3552R Dual Channel, 16-Bit, 33 MUPS, Multispan, Multi-IO SPI DAC Linux device driver <resources/tools-software/linux-drivers/iio-dac/axi-ad3552r>`
+- :dokuwiki:`LTC2387 SAR ADC IIO Linux Driver <resources/tools-software/linux-drivers/iio-adc/ltc2387>`
```

Comment:
@jvelasc2  these should point to docs in https://analogdevicesinc.github.io/hdl/library/index.html
@gastmaier should we point directly to the URL for the core documentation, or is there a role? (Just thinking ahead if stuff moves around)